Завершение локализации разделов "beautifulattempt.sbvn.json" и "bountytarget.config.json"

Munchk1n opened the pull request.

Munchk1n committed:
Непонятен контекст, возможны смысловые ошибки в значении и склонении слова "yelling".

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Слово "stealth" вернее было бы перевести как "скрытность", но вариант "осторожность" в данном случае кажется более подходящим с точки зрения русского языка. Вы когда-либо слышали фразу "подходить к ситуации скрытно"? Я тоже нет.

Munchk1n committed:
Странное построение последнего предложения, при котором крайне проблематично перевести текст в знакомую русскому люду манеру повествования.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Стоит ли выделять "воздушный хоккей" заглавными буквами? Думаю, нет.

Munchk1n committed:
Имеются сомнения насчёт правильности перевода фразы "who instead greets me is a beautiful girl". Её предоставил Google Translator.

Munchk1n committed:
Добавлено дополнительное личное местоимение перед "почти незнакомо" с целью более привычного русскому человеку звучания.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Не уверен насчёт перевода "dream" как "мечта". Вполне возможен вариант "фантазия", имеющий слегка иной смысл.

Munchk1n committed:
"Shrine maidens" дословно можно перевести как "святые девы", но уместно ли назвать их "монахинями" или "жрицами" с целью лаконичности? Эти варианты слишком неоднозначны для принятия одним человеком.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Сомнения насчёт перевода слова "post" как "пост", так как неизвестен контекст. Можно заменить на нейтральное "строение". Также вольно переведены слова "seems low" (дословно "выглядят низкими") как "маловероятны" ради благозвучия.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Исправлены опечатки в словах "благослОвила" и "библиотЕКА". Похоже, после перевода нетронутых фраз стоит заняться проверкой переведённых.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Фраза "skips a beat" ("пропускает удары") заменено на "замирает", что недалеко от сути фразы. Также слово "feels" ("ощущается") заменено на краткое "явно"; впрочем, без обоих можно обойтись.

Munchk1n committed:
Не могу избежать странной формулировки, быть может вы найдёте вариант получше.

Munchk1n committed:
Фраза "second-hand" ("вторые руки") заменена на более нейтральное обозначение, так как "слышать из вторых рук" было бы странной формулировкой.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Слово "raw" можно перевести по-разному, но я остановился на "незрелый". Странный текст...

Munchk1n committed:
Слово "tersely" ("кратко") вольно заменено на "робко".

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
"Кричу", "кричи", или "кричит"?..

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Часть про "низкое трение" кажется странной. Возможно, имеется в виду аэро-хоккей.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Надеюсь, немного рифмы не помешает.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Могут быть иные значения и склонения.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Слово "возникновение", кажется, не очень хорошо подходит.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Сомнение с контекстом развеяно найденным предположительным продолжением диалога.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Внесены небольшие смысловые изменения в виде замены перечисления предметов искусства на их последовательное применение в качестве инструментов "изложения воли". Идентичность оригиналу крайне проблематична.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Не уверен, можно ли перевести "present day" как "день презентации"? Зависит от контекста.

Munchk1n committed:
Обновление translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json

Munchk1n committed:
Надеюсь, я был полезным участником перевода одной из своих любимых игр. Скорее всего буду помогать с локализацией и дальше, так как нет сил ждать полного перевода.

Munchk1n committed:
Обновление translations/texts/interface/cockpit/cockpit.config.json

Munchk1n committed:
Обновление translations/texts/interface/cockpit/cockpit.config.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/bountyboard/bountyboardgui.config.json

Munchk1n committed:
Обновление translations/texts/interface/scripted/bountyboard/bountyboardgui.config.json

Munchk1n committed:
Обновление translations/texts/interface/windowconfig/graphicsmenu.config.json

Munchk1n committed:
Обновление translations/texts/interface/shared_.config.json

Munchk1n committed:
Понятия не имею, что только что перевёл.

Munchk1n committed:
Я считаю, что на качестве локализации скажется адаптация стиля речи флоран с их шипящими "С". И так как при переводе эти "С" оказываются не там, где были раньше, считаю умессстным их переносить.

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Крайне сомнительный перевод, надеюсь на помощь от лучших знатоков инглиша.

Munchk1n committed:
Также сомнительный момент, G-translator даёт совершенно разные формулировки, от "давно я не трепал блондинку", до "давно я не ходил босиком".

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Возникли сильные сомнения насчёт перевода слова "snaunt", он переводится как "нюхать", что-то связанное с носом. Есть идея заменить вариант "нюхач", который ни к чему не отсылает, на "сопли" или "слизь", что будет более логичным, но не передающим истинную суть.

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Очень вольный перевод, вызванный непониманием последней части предложения. "На дне клевания приказа"?..

Munchk1n committed:
Добавлено подведение в виде "с" к скрипту с обозначением банды, которую не удалось одолеть.

Munchk1n committed:
Буквальный перевод, если покажется неказистым можно заменить на "я сейчас очень сожалею (+обо всём*)!"

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Вместо буквального "я знаю, когда проиграл/проигрываю" решил перевести как "я умею проигрывать. Мне кажется, так немного лучше.

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Опустил часть с "than this" ("чем это"), так как плохо вяжется с текстом. Взамен вставил слово "куда" перед "больше".

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Не уверен насчёт перевода "I'll get you back" в качестве "я верну тебя", так как неизвестен контекст. Можно заменить на "я отомщу тебе", но это произвольное.

Munchk1n committed:
Да, странный перевод, но иначе никак, ибо буквально.

Munchk1n committed:
"Сказать" или "передать"?

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

Munchk1n committed:
Обновление translations/texts/dialog/bountytarget.config.json

xomachine commented on translations/texts/dialog/peacekeeper.config.json:
```diff
@@ -154,7 +154,7 @@
     "Texts": {
       "Eng": "asdasd",
-      "Rus": ""
+      "Rus": "асдасд"
```
Если контекст не ясен, лучше оставить без перевода

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1249,7 +1249,7 @@
     "Texts": {
       "Eng": "I open the envelope, my heart racing. Perhaps the letter from Kasumi will instruct me how to return to her time so we can resume our fated love?",
-      "Rus": ""
+      "Rus": "Я открываю конверт, моё сердце бьётся. Возможно, письмо Касуми научит меня, как вернуться к её времени, чтобы мы могли возобновить нашу судьбоносную любовь?"
```
скорее "в её время", поскольку речь идёт о путешествии во времени

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1425,7 +1425,7 @@
     "Texts": {
       "Eng": "Just yelling at everyone!",
-      "Rus": ""
+      "Rus": "Просто кричи на всех!"
```
Не уверен, что тут уместно повелительное наклонение... нужен контекст

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1486,7 +1486,7 @@
     "Texts": {
       "Eng": "My heart races and I feel the world spinning. I reach out but my hands find no solace. I feel like a beached whale, except the beach I'm on is travelling... Travelling through time!",
-      "Rus": ""
+      "Rus": "Моё сердце бьётся, и я чувствую, как вращается мир. Я тянусь, но мои руки не находят утешения. Я чувствую себя китом на берегу, за исключением того, что я путешествую по пляжу... Путешествую во времени!"
```
А вот и нет.
"Я чувствую себя китом, выбросившимся на берег, за исключением того, что мой берег путешествует... путешествует во времени!"

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1668,7 +1672,7 @@
     "Texts": {
       "Eng": "She ignores me entirely, contemplating for a moment before her face lights up.",
-      "Rus": ""
+      "Rus": "Она полностью игнорирует меня, на мгновение обдумывая, как краснеет её лицо."
```
Мимо
Она полностью игнорирует меня, задумавшись на мгновение, перед тем как её лицо покраснело

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1584,31 +1584,35 @@
     "Texts": {
       "Eng": "Practice Air Hockey",
-      "Rus": ""
+      "Rus": "Практикуемся в аэрохоккее."
```
Снова не хватает контекста, похоже это вариант ответа. Возможно даже на вопрос "что ты тут делаешь" или что-то вроде того

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1692,7 +1696,7 @@
     "Texts": {
       "Eng": "She nods. \"I have to come up with a song that will cleanse the land of uncivility. But I will need help coming up with ideas...\"",
-      "Rus": ""
+      "Rus": "Она кивает. \"Я должен придумать песню, которая очистит землю от невежества. Но мне понадобится помощь с возникновением идей...\""
```
Почему она говорит о себе в мужском роде?

xomachine commented on translations/texts/interface/scripted/sbvn/games/beautifulattempt/beautifulattempt.sbvn.json:
```diff
@@ -1865,7 +1869,7 @@
     "Texts": {
       "Eng": "The hockey table... The present day... Kasumi! I feel like fate has been snatched from me.",
-      "Rus": ""
+      "Rus": "Хоккейный стол... Нынешний день... Касуми! Я чувствую как судьба была вырвана из меня."
```
Странное выражение "судьба вырвана из меня"

xomachine commented on translations/texts/interface/shared_.config.json:
```diff
@@ -450,7 +450,8 @@
     "Texts": {
-      "Eng": "x?\\d{0,4}"
+      "Eng": "x?\\d{0,4}",
+      "Rus": "x?\\d{0,4}"
```
не требует перевода (если в следующем релизе разрабы тут что-то изменят мы можем это изменение пропустить)

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -408,7 +408,7 @@
     "Texts": {
       "Eng": "Floran isss proud member of the ^orange;<gangName>^reset;! Floran won't go down easssy!",
-      "Rus": ""
+      "Rus": "Флоран - гордый член ^orange;<gangName>^reset;! Флоран не сдассстся легко!"
```
gangName в именительном падеже, соответственно нужно правильно подстроить предложение

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -656,7 +656,7 @@
     "Texts": {
       "Eng": "Hmph - I let my guard down!",
-      "Rus": ""
+      "Rus": "Хмпф - я позволю своей охране сдаться!"
```
https://idioms.thefreedictionary.com/let+guard+down

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -704,7 +704,7 @@
     "Texts": {
       "Eng": "How did I end up like this?",
-      "Rus": ""
+      "Rus": "Как я закончу, как этот?"
```
Как я дошел до этого?

xomachine reviewed (commented):
Ревью ещё не окончено, в bountytarget закопался. Но кое-что уже можно поправить

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -911,7 +911,7 @@
     "Texts": {
       "Eng": "I don't blame them for running away - they WERE messing with the ^orange;<gangName>^reset;!",
-      "Rus": ""
+      "Rus": "Я не виню их за то, что они сбежали - они ОНИ возились с ^orange;<gangName>^reset;!"
```
Имя банды в именительном падеже, соответственно надо подстроить предложение

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -945,7 +946,8 @@
     "Texts": {
-      "Eng": "I have failed the ^orange;<gangName>^reset;!"
+      "Eng": "I have failed the ^orange;<gangName>^reset;!",
+      "Rus": "Я потерпел неудачу с ^orange;<gangName>^reset;!"
```
Аналогично падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1011,7 +1018,8 @@
     "Texts": {
-      "Eng": "I shall show the true might of the ^orange;<gangName>^reset;!"
+      "Eng": "I shall show the true might of the ^orange;<gangName>^reset;!",
+      "Rus": "Я продемонстрирую истинную мощь ^orange;<gangName>^reset;!"
```
Аналогично падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1036,7 +1045,8 @@
     "Texts": {
-      "Eng": "I suppose they fled. I don't blame them - they WERE messing with the ^orange;<gangName>^reset;!"
+      "Eng": "I suppose they fled. I don't blame them - they WERE messing with the ^orange;<gangName>^reset;!",
+      "Rus": "Я полагаю, они сбежали. Я не виню их, ОНИ возились с ^orange;<gangName>^reset;!"
```
Аналогично падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1047,7 +1057,8 @@
     "Texts": {
-      "Eng": "I thought I was winning..."
+      "Eng": "I thought I was winning...",
+      "Rus": "Я думал, что выиграл..."
```
Я думал, что выигрывал...

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1080,7 +1093,8 @@
     "Texts": {
-      "Eng": "I underestimated you, Peacekeeper. You got me fair and square."
+      "Eng": "I underestimated you, Peacekeeper. You got me fair and square.",
+      "Rus": "Я недооценил тебя, Миротворец. Ты сделал меня честным и справедливым."
```
Нет, got me - значит "сделал меня", но в другом смысле, а именно обставил, победил, но не привил ему честность и справедливость

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1171,7 +1192,8 @@
     "Texts": {
-      "Eng": "I'll be back one day!"
+      "Eng": "I'll be back one day!",
+      "Rus": "Я вернусь в один день!"
```
Однажды я вернусь!

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1205,18 +1229,22 @@
     "Texts": {
-      "Eng": "I'll get you back for this - I never forget a face!"
+      "Eng": "I'll get you back for this - I never forget a face!",
+      "Rus": "Я верну тебя за это - я никогда не забуду твоё лицо!"
```
Куда вернёт?

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1227,7 +1255,8 @@
     "Texts": {
-      "Eng": "I'll get you back some day, Peacekeeper!"
+      "Eng": "I'll get you back some day, Peacekeeper!",
+      "Rus": "Я верну тебя когда-нибудь, Миротворец!"
```
Так куда вернёт-то?

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1252,7 +1282,8 @@
     "Texts": {
-      "Eng": "I'm a member of the ^orange;<gangName>^reset; - couldn't you tell?"
+      "Eng": "I'm a member of the ^orange;<gangName>^reset; - couldn't you tell?",
+      "Rus": "Я член ^orange;<gangName>^reset; - не могли бы вы сказать?"
```
Проблема с падежом имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1310,7 +1345,8 @@
     "Texts": {
-      "Eng": "It ain't gonna be easy to catch a member of the ^orange;<gangName>^reset;!"
+      "Eng": "It ain't gonna be easy to catch a member of the ^orange;<gangName>^reset;!",
+      "Rus": "Не будет легко поймать члена ^orange;<gangName>^reset;!"
```
Проблема с падежом имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1347,7 +1385,8 @@
     "Texts": {
-      "Eng": "It won't be easy to catch a member of the ^orange;<gangName>^reset;!"
+      "Eng": "It won't be easy to catch a member of the ^orange;<gangName>^reset;!",
+      "Rus": "Нелегко поймать члена ^orange;<gangName>^reset;!"
```
Проблема с падежом имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1370,7 +1410,8 @@
     "Texts": {
-      "Eng": "Let us dance!"
+      "Eng": "Let us dance!",
+      "Rus": "Дай нам танцевать!"
```
Станцуем!

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1393,7 +1435,8 @@
     "Texts": {
-      "Eng": "Long live the ^orange;<gangName>^reset;!"
+      "Eng": "Long live the ^orange;<gangName>^reset;!",
+      "Rus": "Долгой жизни ^orange;<gangName>^reset;!"
```
Проблема с падежом имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1404,7 +1447,8 @@
     "Texts": {
-      "Eng": "Looks like I've been wrangled..."
+      "Eng": "Looks like I've been wrangled...",
+      "Rus": "Похоже, я был спорным..."
```
https://www.urbandictionary.com/define.php?term=Wrangled

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1440,7 +1486,8 @@
     "Texts": {
-      "Eng": "My feathers are getting caught in these cuffs! Can you take them off?"
+      "Eng": "My feathers are getting caught in these cuffs! Can you take them off?",
+      "Rus": "Мои перья попадают в эти манжеты! Ты можешь их снять?"
```
cuffs - наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1528,7 +1582,8 @@
     "Texts": {
-      "Eng": "Not like this..."
+      "Eng": "Not like this...",
+      "Rus": "Не так..."
```
Лучше перевести как "Только не это..."

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1539,7 +1594,8 @@
     "Texts": {
-      "Eng": "Of course a FLORAN would be foolish enough to take on the ^orange;<gangName>^reset;!"
+      "Eng": "Of course a FLORAN would be foolish enough to take on the ^orange;<gangName>^reset;!",
+      "Rus": "Конечно, ФЛОРАН был бы глупцом если бы взял на себя ^orange;<gangName>^reset;!"
```
Проблема с падежом имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1564,7 +1621,8 @@
     "Texts": {
-      "Eng": "Okay, okay, you got me!"
+      "Eng": "Okay, okay, you got me!",
+      "Rus": "Хорошо, хорошо, ты взял меня!"
```
ты победил

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1575,7 +1633,8 @@
     "Texts": {
-      "Eng": "One day we shall fight again!"
+      "Eng": "One day we shall fight again!",
+      "Rus": "В один день мы сразимся снова!"
```
однажды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1608,7 +1669,8 @@
     "Texts": {
-      "Eng": "Ouch! Take these cuffs off or you'll regret it!"
+      "Eng": "Ouch! Take these cuffs off or you'll regret it!",
+      "Rus": "Ауч! Сними эти манжеты или ты пожалеешь об этом!"
```
наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1630,7 +1693,8 @@
     "Texts": {
-      "Eng": "Remove these cuffs at once!"
+      "Eng": "Remove these cuffs at once!",
+      "Rus": "Снимите эти манжеты немедленно!"
```
наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1652,7 +1717,8 @@
     "Texts": {
-      "Eng": "Set me free - I promise I'll be good in future! I promise!"
+      "Eng": "Set me free - I promise I'll be good in future! I promise!",
+      "Rus": "Сделай меня свободным - я обещаю быть хорошим в будущем! Я обещаю!"
```
освободи

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1663,7 +1729,8 @@
     "Texts": {
-      "Eng": "So be it..."
+      "Eng": "So be it...",
+      "Rus": "Так будь им..."
```
Да будет так...

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1674,7 +1741,8 @@
     "Texts": {
-      "Eng": "So long, Peacekeeper..."
+      "Eng": "So long, Peacekeeper...",
+      "Rus": "Слишком долго, Миротворец..."
```
Давно не виделись, Миротворец

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1696,7 +1765,8 @@
     "Texts": {
-      "Eng": "So, you want to try again? That's some real monkey business!"
+      "Eng": "So, you want to try again? That's some real monkey business!",
+      "Rus": "И так, ты хочешь попробовать снова? Это реально обезьяний труд!"
```
Есть такой фильм - monkey business, его локализовали как мартышкин труд. Думаю нам тоже стоит переводить так

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1763,7 +1838,8 @@
     "Texts": {
-      "Eng": "Ta-ta, and farewell!"
+      "Eng": "Ta-ta, and farewell!",
+      "Rus": "Та-та, и прощание!"
```
и прощай

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1796,7 +1874,8 @@
     "Texts": {
-      "Eng": "Take these here cuffs off! Now!"
+      "Eng": "Take these here cuffs off! Now!",
+      "Rus": "Сними эти манжеты, сейчас!"
```
наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1841,7 +1923,8 @@
     "Texts": {
-      "Eng": "That's what you get for messin' with the ^orange;<gangName>^reset;!"
+      "Eng": "That's what you get for messin' with the ^orange;<gangName>^reset;!",
+      "Rus": "Вот что ты получаешь за то, что связался с ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1855,7 +1938,8 @@
     "Texts": {
-      "Eng": "That's what you get for messing with the ^orange;<gangName>^reset;!"
+      "Eng": "That's what you get for messing with the ^orange;<gangName>^reset;!",
+      "Rus": "Вот что ты получаешь за то, что связался с ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1869,7 +1953,8 @@
     "Texts": {
-      "Eng": "That's what you get for trying to lock ME up!"
+      "Eng": "That's what you get for trying to lock ME up!",
+      "Rus": "Вот что ты получаеешь за попытку запереть меня!"
```
получаеешь

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1880,7 +1965,8 @@
     "Texts": {
-      "Eng": "The Miniknog can't catch me - what makes you think you can?"
+      "Eng": "The Miniknog can't catch me - what makes you think you can?",
+      "Rus": "Миникног не может поймать меня - с чего ты думаешь, что сможешь?"
```
Miniknog - это Министерство знаний - Ministery of Knowlege - сокращено на манер новояза из 1984. В нашем переводе мы употребляем просто Министерство, так как вразумительных альтернатив придумать не удалось (не Минзнай же). См. Глоссарий

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1891,7 +1977,8 @@
     "Texts": {
-      "Eng": "The ^orange;<gangName>^reset; ain't gonna stand for this!"
+      "Eng": "The ^orange;<gangName>^reset; ain't gonna stand for this!",
+      "Rus": "^orange;<gangName>^reset; не стоит за этим!"
```
Имя банды часто во множественном числе, но лучше перефразировать чтобы число и падеж не играли роли

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -1902,7 +1989,8 @@
     "Texts": {
-      "Eng": "The ^orange;<gangName>^reset; are sick of nosey folks like you!"
+      "Eng": "The ^orange;<gangName>^reset; are sick of nosey folks like you!",
+      "Rus": "^orange;<gangName>^reset; надоели любопытные люди, как ты!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2023,7 +2121,8 @@
     "Texts": {
-      "Eng": "The ^orange;<gangName>^reset; will not be bested!"
+      "Eng": "The ^orange;<gangName>^reset; will not be bested!",
+      "Rus": "^orange;<gangName>^reset; не будет лучше!"
```
to be bested - быть превзойдённым, тем кого превзошли

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2034,7 +2133,8 @@
     "Texts": {
-      "Eng": "The ^orange;<gangName>^reset; won't let this slide, Peacekeeper!"
+      "Eng": "The ^orange;<gangName>^reset; won't let this slide, Peacekeeper!",
+      "Rus": "^orange;<gangName>^reset; не пропусти этот слайд, Миротворец!"
```
не спустят тебе этого с рук

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2045,7 +2145,8 @@
     "Texts": {
-      "Eng": "The ^orange;<gangName>^reset; won't stand for this!"
+      "Eng": "The ^orange;<gangName>^reset; won't stand for this!",
+      "Rus": "^orange;<gangName>^reset; не будут за это!"
```
не потерпят этого

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2067,7 +2169,8 @@
     "Texts": {
-      "Eng": "These cuffs are a little too tight!"
+      "Eng": "These cuffs are a little too tight!",
+      "Rus": "Эти манжеты слишком тесные!"
```
наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2078,7 +2181,8 @@
     "Texts": {
-      "Eng": "These cuffs are too tight - they are going to leave dents!"
+      "Eng": "These cuffs are too tight - they are going to leave dents!",
+      "Rus": "Эти манжеты очень тесные - они оставят следы!"
```
наручники

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2100,7 +2205,8 @@
     "Texts": {
-      "Eng": "These handcuffs aren't my size! You should probably take them off me."
+      "Eng": "These handcuffs aren't my size! You should probably take them off me.",
+      "Rus": "Эти наручники не моего размера! Вы, вероятны, должны снять их с меня."
```
вероятно

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2166,7 +2277,8 @@
     "Texts": {
-      "Eng": "This is what you get for messing with the ^orange;<gangName>^reset;!"
+      "Eng": "This is what you get for messing with the ^orange;<gangName>^reset;!",
+      "Rus": "Это то, что ты получаешь за связь с ^orange;<gangName>^reset;!"
```
за то что связываешься с бандой ...

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2378,7 +2508,8 @@
     "Texts": {
-      "Eng": "Yet another win for the ^orange;<gangName>^reset;!"
+      "Eng": "Yet another win for the ^orange;<gangName>^reset;!",
+      "Rus": "Ещё одна победа для ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2433,7 +2568,8 @@
     "Texts": {
-      "Eng": "You ain't smart if yer thinkin' o' challengin' the ^orange;<gangName>^reset;!"
+      "Eng": "You ain't smart if yer thinkin' o' challengin' the ^orange;<gangName>^reset;!",
+      "Rus": "Ты не очень умён, если бросаешь вызов ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2444,7 +2580,8 @@
     "Texts": {
-      "Eng": "You are a fool to cross blades with the ^orange;<gangName>^reset;!"
+      "Eng": "You are a fool to cross blades with the ^orange;<gangName>^reset;!",
+      "Rus": "Ты глупец, если собираешься скрестить лезвия с ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2470,7 +2608,8 @@
     "Texts": {
-      "Eng": "You came back for another round against the ^orange;<gangName>^reset;?"
+      "Eng": "You came back for another round against the ^orange;<gangName>^reset;?",
+      "Rus": "Ты вернулся за ещё одним раундом против ^orange;<gangName>^reset;?"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2481,7 +2620,8 @@
     "Texts": {
-      "Eng": "You came back for another round against the ^orange;<gangName>^reset;? I guess you ain't very smart!"
+      "Eng": "You came back for another round against the ^orange;<gangName>^reset;? I guess you ain't very smart!",
+      "Rus": "Ты вернулся за ещё одним раундом против ^orange;<gangName>^reset;? Я думаю, ты не очень умён!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2543,7 +2687,8 @@
     "Texts": {
-      "Eng": "You cannot defeat a member of the ^orange;<gangName>^reset;!"
+      "Eng": "You cannot defeat a member of the ^orange;<gangName>^reset;!",
+      "Rus": "Ты не сможешь одолеть члена ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2580,7 +2727,8 @@
     "Texts": {
-      "Eng": "You have made a grave mistake by challenging the ^orange;<gangName>^reset;!"
+      "Eng": "You have made a grave mistake by challenging the ^orange;<gangName>^reset;!",
+      "Rus": "Ты совершил роковую ошибку, бросив вызов ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2628,7 +2779,8 @@
     "Texts": {
-      "Eng": "You messsed with the wrong gang! Floran isss part of the ^orange;<gangName>^reset;!"
+      "Eng": "You messsed with the wrong gang! Floran isss part of the ^orange;<gangName>^reset;!",
+      "Rus": "Ты связался не с той бандой! Флоран часссть ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2639,7 +2791,8 @@
     "Texts": {
-      "Eng": "You really made a monkey out of me..."
+      "Eng": "You really made a monkey out of me...",
+      "Rus": "Ты действительно сделал из меня обезьяну..."
```
https://idioms.thefreedictionary.com/make+a+monkey+out+of+me

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2661,7 +2815,8 @@
     "Texts": {
-      "Eng": "You think you can ssstop the ^orange;<gangName>^reset;?"
+      "Eng": "You think you can ssstop the ^orange;<gangName>^reset;?",
+      "Rus": "Ты думаешь, что сможешь оссстановить ^orange;<gangName>^reset;?"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2694,7 +2851,8 @@
     "Texts": {
-      "Eng": "You'll never stop the ^orange;<gangName>^reset;!"
+      "Eng": "You'll never stop the ^orange;<gangName>^reset;!",
+      "Rus": "Ты никогда не остановишь ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2737,7 +2897,8 @@
     "Texts": {
-      "Eng": "You're gonna wish you didn't come after the ^orange;<gangName>^reset;!"
+      "Eng": "You're gonna wish you didn't come after the ^orange;<gangName>^reset;!",
+      "Rus": "Ты можешь не вернуться после встречи с ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine commented on translations/texts/dialog/bountytarget.config.json:
```diff
@@ -2777,7 +2940,8 @@
     "Texts": {
-      "Eng": "You've made a mistake messing with the ^orange;<gangName>^reset;!"
+      "Eng": "You've made a mistake messing with the ^orange;<gangName>^reset;!",
+      "Rus": "Ты совершил ошибку, связавшись с ^orange;<gangName>^reset;!"
```
Не учтён падеж имени банды

xomachine reviewed (commented):
Очень много где не учтён падеж для банды
эти предложения при подстановке будут звучать глупо, например:
Ты совершил ошибку, связавшись с Стальные парни!

Чтобы этого избежать советую поправить на 
Ты совершил ошибку, связавшись с бандой Стальные парни!
Это нужно сделать во всех местах где я отметил выше.
В остальном перевод неплохой, очень надеюсь на исправления!

Munchk1n commented:
Все огрехи усвоил, будут исправлены. Есть ли возможность быстрой и точечной
поправки ошибок без необходимости пролистывать файлы заново?
Зарегистрировался на сайте чтобы внести свой вклад в перевод, пока вообще в
нём не разбираюсь.

пт, 25 окт. 2019 г. в 21:53, Dmitriy Fomichev <notifications@github.com>:

> *@xomachine* commented on this pull request.
>
> Очень много где не учтён падеж для банды
> эти предложения при подстановке будут звучать глупо, например:
> Ты совершил ошибку, связавшись с Стальные парни!
>
> Чтобы этого избежать советую поправить на
> Ты совершил ошибку, связавшись с бандой Стальные парни!
> Это нужно сделать во всех местах где я отметил выше.
> В остальном перевод неплохой, очень надеюсь на исправления!
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339172193>
> :
>
> > @@ -911,7 +911,7 @@
>      },
>      "Texts": {
>        "Eng": "I don't blame them for running away - they WERE messing with the ^orange;<gangName>^reset;!",
> -      "Rus": ""
> +      "Rus": "Я не виню их за то, что они сбежали - они ОНИ возились с ^orange;<gangName>^reset;!"
>
> Имя банды в именительном падеже, соответственно надо подстроить предложение
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339173007>
> :
>
> > @@ -945,7 +946,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I have failed the ^orange;<gangName>^reset;!"
> +      "Eng": "I have failed the ^orange;<gangName>^reset;!",
> +      "Rus": "Я потерпел неудачу с ^orange;<gangName>^reset;!"
>
> Аналогично падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339173501>
> :
>
> > @@ -1011,7 +1018,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I shall show the true might of the ^orange;<gangName>^reset;!"
> +      "Eng": "I shall show the true might of the ^orange;<gangName>^reset;!",
> +      "Rus": "Я продемонстрирую истинную мощь ^orange;<gangName>^reset;!"
>
> Аналогично падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339173608>
> :
>
> > @@ -1036,7 +1045,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I suppose they fled. I don't blame them - they WERE messing with the ^orange;<gangName>^reset;!"
> +      "Eng": "I suppose they fled. I don't blame them - they WERE messing with the ^orange;<gangName>^reset;!",
> +      "Rus": "Я полагаю, они сбежали. Я не виню их, ОНИ возились с ^orange;<gangName>^reset;!"
>
> Аналогично падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339173772>
> :
>
> > @@ -1047,7 +1057,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I thought I was winning..."
> +      "Eng": "I thought I was winning...",
> +      "Rus": "Я думал, что выиграл..."
>
> Я думал, что выигрывал...
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339174517>
> :
>
> > @@ -1080,7 +1093,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I underestimated you, Peacekeeper. You got me fair and square."
> +      "Eng": "I underestimated you, Peacekeeper. You got me fair and square.",
> +      "Rus": "Я недооценил тебя, Миротворец. Ты сделал меня честным и справедливым."
>
> Нет, got me - значит "сделал меня", но в другом смысле, а именно обставил,
> победил, но не привил ему честность и справедливость
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175133>
> :
>
> > @@ -1171,7 +1192,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I'll be back one day!"
> +      "Eng": "I'll be back one day!",
> +      "Rus": "Я вернусь в один день!"
>
> Однажды я вернусь!
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175339>
> :
>
> >      "Files": {
>        "dialog/bountytarget.config": [
>          "/completeArrest/default/default/7"
>        ]
>      },
>      "Texts": {
> -      "Eng": "I'll get you back for this - I never forget a face!"
> +      "Eng": "I'll get you back for this - I never forget a face!",
> +      "Rus": "Я верну тебя за это - я никогда не забуду твоё лицо!"
>
> Куда вернёт?
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175461>
> :
>
> > @@ -1227,7 +1255,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I'll get you back some day, Peacekeeper!"
> +      "Eng": "I'll get you back some day, Peacekeeper!",
> +      "Rus": "Я верну тебя когда-нибудь, Миротворец!"
>
> Так куда вернёт-то?
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175682>
> :
>
> > @@ -1252,7 +1282,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "I'm a member of the ^orange;<gangName>^reset; - couldn't you tell?"
> +      "Eng": "I'm a member of the ^orange;<gangName>^reset; - couldn't you tell?",
> +      "Rus": "Я член ^orange;<gangName>^reset; - не могли бы вы сказать?"
>
> Проблема с падежом имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175827>
> :
>
> > @@ -1310,7 +1345,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "It ain't gonna be easy to catch a member of the ^orange;<gangName>^reset;!"
> +      "Eng": "It ain't gonna be easy to catch a member of the ^orange;<gangName>^reset;!",
> +      "Rus": "Не будет легко поймать члена ^orange;<gangName>^reset;!"
>
> Проблема с падежом имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339175920>
> :
>
> > @@ -1347,7 +1385,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "It won't be easy to catch a member of the ^orange;<gangName>^reset;!"
> +      "Eng": "It won't be easy to catch a member of the ^orange;<gangName>^reset;!",
> +      "Rus": "Нелегко поймать члена ^orange;<gangName>^reset;!"
>
> Проблема с падежом имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339176046>
> :
>
> > @@ -1370,7 +1410,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Let us dance!"
> +      "Eng": "Let us dance!",
> +      "Rus": "Дай нам танцевать!"
>
> Станцуем!
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339176336>
> :
>
> > @@ -1393,7 +1435,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Long live the ^orange;<gangName>^reset;!"
> +      "Eng": "Long live the ^orange;<gangName>^reset;!",
> +      "Rus": "Долгой жизни ^orange;<gangName>^reset;!"
>
> Проблема с падежом имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339177265>
> :
>
> > @@ -1404,7 +1447,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Looks like I've been wrangled..."
> +      "Eng": "Looks like I've been wrangled...",
> +      "Rus": "Похоже, я был спорным..."
>
> https://www.urbandictionary.com/define.php?term=Wrangled
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339177547>
> :
>
> > @@ -1440,7 +1486,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "My feathers are getting caught in these cuffs! Can you take them off?"
> +      "Eng": "My feathers are getting caught in these cuffs! Can you take them off?",
> +      "Rus": "Мои перья попадают в эти манжеты! Ты можешь их снять?"
>
> cuffs - наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339177972>
> :
>
> > @@ -1528,7 +1582,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Not like this..."
> +      "Eng": "Not like this...",
> +      "Rus": "Не так..."
>
> Лучше перевести как "Только не это..."
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339178153>
> :
>
> > @@ -1539,7 +1594,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Of course a FLORAN would be foolish enough to take on the ^orange;<gangName>^reset;!"
> +      "Eng": "Of course a FLORAN would be foolish enough to take on the ^orange;<gangName>^reset;!",
> +      "Rus": "Конечно, ФЛОРАН был бы глупцом если бы взял на себя ^orange;<gangName>^reset;!"
>
> Проблема с падежом имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339178288>
> :
>
> > @@ -1564,7 +1621,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Okay, okay, you got me!"
> +      "Eng": "Okay, okay, you got me!",
> +      "Rus": "Хорошо, хорошо, ты взял меня!"
>
> ты победил
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339178351>
> :
>
> > @@ -1575,7 +1633,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "One day we shall fight again!"
> +      "Eng": "One day we shall fight again!",
> +      "Rus": "В один день мы сразимся снова!"
>
> однажды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339178905>
> :
>
> > @@ -1608,7 +1669,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Ouch! Take these cuffs off or you'll regret it!"
> +      "Eng": "Ouch! Take these cuffs off or you'll regret it!",
> +      "Rus": "Ауч! Сними эти манжеты или ты пожалеешь об этом!"
>
> наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339178976>
> :
>
> > @@ -1630,7 +1693,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Remove these cuffs at once!"
> +      "Eng": "Remove these cuffs at once!",
> +      "Rus": "Снимите эти манжеты немедленно!"
>
> наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339179059>
> :
>
> > @@ -1652,7 +1717,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Set me free - I promise I'll be good in future! I promise!"
> +      "Eng": "Set me free - I promise I'll be good in future! I promise!",
> +      "Rus": "Сделай меня свободным - я обещаю быть хорошим в будущем! Я обещаю!"
>
> освободи
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339179113>
> :
>
> > @@ -1663,7 +1729,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "So be it..."
> +      "Eng": "So be it...",
> +      "Rus": "Так будь им..."
>
> Да будет так...
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339179279>
> :
>
> > @@ -1674,7 +1741,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "So long, Peacekeeper..."
> +      "Eng": "So long, Peacekeeper...",
> +      "Rus": "Слишком долго, Миротворец..."
>
> Давно не виделись, Миротворец
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339187180>
> :
>
> > @@ -1696,7 +1765,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "So, you want to try again? That's some real monkey business!"
> +      "Eng": "So, you want to try again? That's some real monkey business!",
> +      "Rus": "И так, ты хочешь попробовать снова? Это реально обезьяний труд!"
>
> Есть такой фильм - monkey business, его локализовали как мартышкин труд.
> Думаю нам тоже стоит переводить так
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339187624>
> :
>
> > @@ -1763,7 +1838,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Ta-ta, and farewell!"
> +      "Eng": "Ta-ta, and farewell!",
> +      "Rus": "Та-та, и прощание!"
>
> и прощай
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339187710>
> :
>
> > @@ -1796,7 +1874,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Take these here cuffs off! Now!"
> +      "Eng": "Take these here cuffs off! Now!",
> +      "Rus": "Сними эти манжеты, сейчас!"
>
> наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339187886>
> :
>
> > @@ -1841,7 +1923,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "That's what you get for messin' with the ^orange;<gangName>^reset;!"
> +      "Eng": "That's what you get for messin' with the ^orange;<gangName>^reset;!",
> +      "Rus": "Вот что ты получаешь за то, что связался с ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339187943>
> :
>
> > @@ -1855,7 +1938,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "That's what you get for messing with the ^orange;<gangName>^reset;!"
> +      "Eng": "That's what you get for messing with the ^orange;<gangName>^reset;!",
> +      "Rus": "Вот что ты получаешь за то, что связался с ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339188033>
> :
>
> > @@ -1869,7 +1953,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "That's what you get for trying to lock ME up!"
> +      "Eng": "That's what you get for trying to lock ME up!",
> +      "Rus": "Вот что ты получаеешь за попытку запереть меня!"
>
> получаеешь
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339188881>
> :
>
> > @@ -1880,7 +1965,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The Miniknog can't catch me - what makes you think you can?"
> +      "Eng": "The Miniknog can't catch me - what makes you think you can?",
> +      "Rus": "Миникног не может поймать меня - с чего ты думаешь, что сможешь?"
>
> Miniknog - это Министерство знаний - Ministery of Knowlege - сокращено на
> манер новояза из 1984. В нашем переводе мы употребляем просто Министерство,
> так как вразумительных альтернатив придумать не удалось (не Минзнай же).
> См. Глоссарий
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189090>
> :
>
> > @@ -1891,7 +1977,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The ^orange;<gangName>^reset; ain't gonna stand for this!"
> +      "Eng": "The ^orange;<gangName>^reset; ain't gonna stand for this!",
> +      "Rus": "^orange;<gangName>^reset; не стоит за этим!"
>
> Имя банды часто во множественном числе, но лучше перефразировать чтобы
> число и падеж не играли роли
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189195>
> :
>
> > @@ -1902,7 +1989,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The ^orange;<gangName>^reset; are sick of nosey folks like you!"
> +      "Eng": "The ^orange;<gangName>^reset; are sick of nosey folks like you!",
> +      "Rus": "^orange;<gangName>^reset; надоели любопытные люди, как ты!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189581>
> :
>
> > @@ -2023,7 +2121,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The ^orange;<gangName>^reset; will not be bested!"
> +      "Eng": "The ^orange;<gangName>^reset; will not be bested!",
> +      "Rus": "^orange;<gangName>^reset; не будет лучше!"
>
> to be bested - быть превзойдённым, тем кого превзошли
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189772>
> :
>
> > @@ -2034,7 +2133,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The ^orange;<gangName>^reset; won't let this slide, Peacekeeper!"
> +      "Eng": "The ^orange;<gangName>^reset; won't let this slide, Peacekeeper!",
> +      "Rus": "^orange;<gangName>^reset; не пропусти этот слайд, Миротворец!"
>
> не спустят тебе этого с рук
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189855>
> :
>
> > @@ -2045,7 +2145,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "The ^orange;<gangName>^reset; won't stand for this!"
> +      "Eng": "The ^orange;<gangName>^reset; won't stand for this!",
> +      "Rus": "^orange;<gangName>^reset; не будут за это!"
>
> не потерпят этого
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189935>
> :
>
> > @@ -2067,7 +2169,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "These cuffs are a little too tight!"
> +      "Eng": "These cuffs are a little too tight!",
> +      "Rus": "Эти манжеты слишком тесные!"
>
> наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339189970>
> :
>
> > @@ -2078,7 +2181,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "These cuffs are too tight - they are going to leave dents!"
> +      "Eng": "These cuffs are too tight - they are going to leave dents!",
> +      "Rus": "Эти манжеты очень тесные - они оставят следы!"
>
> наручники
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190073>
> :
>
> > @@ -2100,7 +2205,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "These handcuffs aren't my size! You should probably take them off me."
> +      "Eng": "These handcuffs aren't my size! You should probably take them off me.",
> +      "Rus": "Эти наручники не моего размера! Вы, вероятны, должны снять их с меня."
>
> вероятно
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190344>
> :
>
> > @@ -2166,7 +2277,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "This is what you get for messing with the ^orange;<gangName>^reset;!"
> +      "Eng": "This is what you get for messing with the ^orange;<gangName>^reset;!",
> +      "Rus": "Это то, что ты получаешь за связь с ^orange;<gangName>^reset;!"
>
> за то что связываешься с бандой ...
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190758>
> :
>
> > @@ -2378,7 +2508,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "Yet another win for the ^orange;<gangName>^reset;!"
> +      "Eng": "Yet another win for the ^orange;<gangName>^reset;!",
> +      "Rus": "Ещё одна победа для ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190879>
> :
>
> > @@ -2433,7 +2568,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You ain't smart if yer thinkin' o' challengin' the ^orange;<gangName>^reset;!"
> +      "Eng": "You ain't smart if yer thinkin' o' challengin' the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты не очень умён, если бросаешь вызов ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190915>
> :
>
> > @@ -2444,7 +2580,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You are a fool to cross blades with the ^orange;<gangName>^reset;!"
> +      "Eng": "You are a fool to cross blades with the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты глупец, если собираешься скрестить лезвия с ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339190973>
> :
>
> > @@ -2470,7 +2608,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You came back for another round against the ^orange;<gangName>^reset;?"
> +      "Eng": "You came back for another round against the ^orange;<gangName>^reset;?",
> +      "Rus": "Ты вернулся за ещё одним раундом против ^orange;<gangName>^reset;?"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191023>
> :
>
> > @@ -2481,7 +2620,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You came back for another round against the ^orange;<gangName>^reset;? I guess you ain't very smart!"
> +      "Eng": "You came back for another round against the ^orange;<gangName>^reset;? I guess you ain't very smart!",
> +      "Rus": "Ты вернулся за ещё одним раундом против ^orange;<gangName>^reset;? Я думаю, ты не очень умён!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191136>
> :
>
> > @@ -2543,7 +2687,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You cannot defeat a member of the ^orange;<gangName>^reset;!"
> +      "Eng": "You cannot defeat a member of the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты не сможешь одолеть члена ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191220>
> :
>
> > @@ -2580,7 +2727,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You have made a grave mistake by challenging the ^orange;<gangName>^reset;!"
> +      "Eng": "You have made a grave mistake by challenging the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты совершил роковую ошибку, бросив вызов ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191354>
> :
>
> > @@ -2628,7 +2779,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You messsed with the wrong gang! Floran isss part of the ^orange;<gangName>^reset;!"
> +      "Eng": "You messsed with the wrong gang! Floran isss part of the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты связался не с той бандой! Флоран часссть ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191629>
> :
>
> > @@ -2639,7 +2791,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You really made a monkey out of me..."
> +      "Eng": "You really made a monkey out of me...",
> +      "Rus": "Ты действительно сделал из меня обезьяну..."
>
> https://idioms.thefreedictionary.com/make+a+monkey+out+of+me
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191708>
> :
>
> > @@ -2661,7 +2815,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You think you can ssstop the ^orange;<gangName>^reset;?"
> +      "Eng": "You think you can ssstop the ^orange;<gangName>^reset;?",
> +      "Rus": "Ты думаешь, что сможешь оссстановить ^orange;<gangName>^reset;?"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191755>
> :
>
> > @@ -2694,7 +2851,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You'll never stop the ^orange;<gangName>^reset;!"
> +      "Eng": "You'll never stop the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты никогда не остановишь ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191846>
> :
>
> > @@ -2737,7 +2897,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You're gonna wish you didn't come after the ^orange;<gangName>^reset;!"
> +      "Eng": "You're gonna wish you didn't come after the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты можешь не вернуться после встречи с ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
> ------------------------------
>
> In translations/texts/dialog/bountytarget.config.json
> <https://github.com/SBT-community/Starbound_RU/pull/114#discussion_r339191917>
> :
>
> > @@ -2777,7 +2940,8 @@
>        ]
>      },
>      "Texts": {
> -      "Eng": "You've made a mistake messing with the ^orange;<gangName>^reset;!"
> +      "Eng": "You've made a mistake messing with the ^orange;<gangName>^reset;!",
> +      "Rus": "Ты совершил ошибку, связавшись с ^orange;<gangName>^reset;!"
>
> Не учтён падеж имени банды
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SBT-community/Starbound_RU/pull/114?email_source=notifications&email_token=ANSNZ5CHV7EQLBPF6E7MDKTQQM6AZA5CNFSM4JEOOFM2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCJI757Y#pullrequestreview-307363583>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANSNZ5ANXETPQKLB63GOPJDQQM6AZANCNFSM4JEOOFMQ>
> .
>

xomachine commented:
Можно редактировать прямо тут в интерфейсе гитхаба.

Нужно перейти во вкладку Files changed в этом реквесте, нажать на 3 точки и выбрать Edit file
Пролистывать файл придётся (хотя можно ориентироваться по номерам строк), но вот искать его в переводе уже будет не нужно.

Как коментарии из реквестов отображать в удобном виде в интерфейсе переводчика я пока не придумал, к сожалению

Munchk1n committed:
Update peacekeeper.config.json

Munchk1n committed:
Update beautifulattempt.sbvn.json

Munchk1n committed:
Update bountytarget.config.json

Исправил указанные логические ошибки и опечатки, а также чуть сверху. Оставил нетронутым перевод "you got me", так как сомневаюсь в верности перевода как "ты победил". В моём понимании, наш "Миротворец" после победы именно ловит их, доставляя в космическую тюрьму; в общем, на Ваш суд. Также я так и не узнал правильный перевод слова "wrangled", так что его тоже не трогал. Остальные строки, похоже, наконец доведены до ума и благозвучия.

xomachine committed:
Мелкие исправление

xomachine committed:
Убрал непонятный перевод

xomachine committed:
Ещё один мелкий недочёт

xomachine committed:
Update bountytarget.config.json

xomachine commented:
Думаю, в таком виде перевод можно принять.
Спасибо за проделанную работу!
